SimPEG/utils/io_utils/io_utils_electromagnetics.py

```diff
@@ -78,17 +181,205 @@ def read_dcipoctree_ubc(file_name, data_type):
         defined as uncertainties.
 
     """
+    data_type = data_type.lower()
+    format_type = format_type.lower()
 
-    # Unused for now but it will be when we manage IP types better.
-    assert data_type.lower() in [
+    assert data_type in [
```

Comment:
Since you're changing this line anyways, again do not use `assert`, instead throw the appropriate error (`ValueError`), also keep the `.lower()` part.

Reply:
nvm on the `.lower()` part, didn't see it a few lines up